### ISSUE TITLE
[Improvement] Destructuring improvement

### ISSUE THREAD

**Issue by da411d** [javascript-obfuscator/javascript-obfuscator#688]
Destructured variables are not renamed and it is bad because code becomes easier to understand. I think it will be better to [assign them to new variable](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Assigning_to_new_variable_names)

Example code:
```
const { getSecret, getSuperSecret } = require("./secret");

const secret = getSecret();
const superSecret = getSuperSecret();

console.log(secret, superSecret);
```

Current output:
```
const {getSecret,getSuperSecret}=require(r('0x2')+r('0x0')),P=getSecret(),n=getSuperSecret();console[r('0x1')](P,n);
```

My way to fix:
```
const {getSecret:v,getSuperSecret:E}=require(r('0x2')+r('0x0')),P=v(),n=E();console[r('0x1')](P,n);
```

**Comment by sanex3339** [comment]
Good idea and simple to implement

**Comment by sanex3339** [comment]
Merged to the `master` I'll release a new with these changes this weekend.

**Comment by sanex3339** [comment]
Released as `1.9.0`

**Comment by da411d** [comment]
One more improvement idea:
```
const keyA = "getSecret", keyB = "getSuperSecret"; // get from tring array or somewhere else
const {[keyA]:v,[keyB]:E}=require(r('0x2')+r('0x0')),P=v(),n=E();console[r('0x1')](P,n);
```
